backend/internal/adapters/secondary/repositories/gem.go

```diff
@@ -51,18 +51,16 @@ func (r *gemRepository) GetPendingCollectionByUser(userId int64) ([]gems.Gem, er
 	return gems, nil
 }
 
-func (r *gemRepository) Add(message string) (*gems.Gem, error) {
-	sql, _, _ := qb.Insert("gems").Rows(goqu.Record{
-		"message": message,
-	}).Returning(goqu.T("gems").All()).ToSQL()
+func (r *gemRepository) Add(gem *gems.Gem) (string, error) {
+	sql, _, _ := qb.Insert("gems").ToSQL()
```

Comment:
hmm, does this work? I don't think this statement actually inserts the gem haha

Reply:
HAHA oops, updated